packages/astro/src/template/5xx.ts

```diff
@@ -1,80 +0,0 @@
-import { encode } from 'html-entities';
-import { baseCSS } from './css.js';
-
-interface ErrorTemplateOptions {
-	/** a short description of the error */
-	message: string;
-	/** information about where the error occurred */
-	stack?: string;
-	/** HTTP error code */
-	statusCode?: number;
-	/** HTML <title> */
-	tabTitle: string;
-	/** page title */
-	title: string;
-	/** show user a URL for more info or action to take */
-	url?: string;
-}
-
-/** Display all errors */
-export default function template({
```

Comment:
don't we still need a 500 template in case the server logic fails on a fresh page load? Or does Vite handle this for us now.